### ISSUE TITLE
Autobot Autoupdater

### ISSUE THREAD

**Issue by SpadesNeil** [LCN-Dev/GoogleBot#113]
#### Brief description of the issue:
We need to get the bot to automatically check for a new version after each call by seeing if there's a newer file available on the web server that we use for soundboard updating.

If a new version exists, pause the current bot, download the new version, close down all Chrome windows, launch the _new_ bot, and close the current bot, in that order.

Required will also be an auto-start feature that automatically connects to the dialer if certain command line conditions are used during the bot startup. These command lines will be used by the bot to tell it to auto-start its self.

#### Additional info:
 - **Bot Revision**: https://github.com/LCN-Dev/GoogleBot/commit/a8b2a096207e1b4506fec1e726ac31605d78d541

**Comment by MTS11648** [comment]
Was working on something like this, but the way I was doing it was kind of impractical. Is there any way we can use that Box program to accomplish something like this?

**Comment by SpadesNeil** [comment]
No, but there may be a better way, using a method similar to how the bot updates its soundboard.

The trick is making it so the old bot starts the new bot using a command line argument (-autostart) so the new bot knows to auto-start its self when it's run.

Video games often use command lines in order to tell the game to start with certain mods or debug options enabled. We can probably do the same thing here.
